src/main/java/uk/gov/hmcts/reform/judicialapi/configuration/FeignInterceptorConfiguration.java

```diff
@@ -49,6 +50,8 @@ public RequestInterceptor requestInterceptor(FeignHeaderConfig config) {
                     log.warn("{}:: FeignHeadConfiguration {}", loggingComponentName, "Failed to get request header!");
                 }
             }
+            requestTemplate.header(RefDataConstants.SERVICE_AUTHORIZATION,
```

Comment:
this can be removed unless you are creating LRD S2S and passing to LRD without whitelisting JRD in LRD